ESPixelStick/ESPixelStick.ino

```diff
@@ -201,6 +204,11 @@ void setup()
     SensorDS18B20.Begin();
 #endif // def SUPPORT_SENSOR_DS18B20
 
+#ifdef USE_OLED
+    OLED.Begin();
```

Comment:
If these two steps are done at the same time, then please combine them in the library. Do not drag library specific functionality into the general purpose code (this is an area we are working on cleaning up).